lib/cli.js

```diff
@@ -9,8 +9,19 @@ const configuration = getConfig();
 const choicesList = choices(configuration);
 const questionsList = questions(choicesList);
 
-inquirer.prompt(questionsList).then((answers) => {
-  const message = answers.moreInfo ? `${answers.editor}` : `${answers.type} ${answers.description}`;
+const prompt = inquirer.prompt(questionsList);
+
+prompt.ui.process.subscribe((answer) => {
+  if (answer.name === 'commitMessageWarning') {
+    const continueExecution = answer.answer;
+    if (!continueExecution) {
+      process.exit();
```

Comment:
Not with the current structure, which utilizes the _questions_ object. We'd need to do it somehow [using these features](https://github.com/SBoudrias/Inquirer.js/#reactive).
So, either
- we enforce the rule like 100%, which means we would keep the structure as it's currently on master.
- we show a warning, and if the user says he wants to cancel, the whole system stops, as it's done in this pull request
- or we rebuild it in yet another way, so we can ask questions more dynamically.